pkg/simple/client/devops/jenkins/pipeline.go

```diff
@@ -170,34 +171,88 @@ func (p *Pipeline) GetPipelineRun() (*devops.PipelineRun, error) {
 }
 
 func (p *Pipeline) ListPipelineRuns() (*devops.PipelineRunList, error) {
-	res, err := p.Jenkins.SendPureRequest(p.Path, p.HttpParameters)
+	// please don't remove below code lines until blueocen fixed
+	//res, err := p.Jenkins.SendPureRequest(p.Path, p.HttpParameters)
```

Comment:
Please remove the unused code. If it's commented, it's unnecessary, right?

Reply:
As I said in the comment, I prefer to keep it.

Reply:
It's unprofessional, you can file another PR when blueocean fix it.

Reply:
How about moving those code lines to a new function?

Reply:
yeah, that's ok

Reply:
done.